### DISCUSSION TITLE
How to set a background color for images?

### DISCUSSION THREAD

**Discussion by mynona** [trvswgnr/bs5-lightbox#43] [Q&A] [answered]
How to set a background color for images? Use case: I have transparent pngs but would like to display them with a predefined background in the lightbox.

the black bars should be transparent

**Comment by createperhaps** [comment, marked as answer]
Did you find a solution?

If it is the same issue I am having I used

`.ratio {
            background-color: transparent !important;
        }`

but I am wondering if there's another way.
